From PeterHuewe@gmx.de  Mon Oct 26 14:32:07 2009
From: Peter Hüwe <PeterHuewe@gmx.de>
Date: Tue, 29 Sep 2009 01:49:00 +0200
Subject: Staging: p9auth:  adding __init/__exit macros
To: "Greg Kroah-Hartman" <gregkh@suse.de>
Cc: "Serge E. Hallyn" <serue@us.ibm.com>, devel@driverdev.osuosl.org, linux-kernel@vger.kernel.org, Jiri Kosina <trivial@kernel.org>, kernel-janitors@vger.kernel.org
Message-ID: <200909290149.01257.PeterHuewe@gmx.de>
Content-Disposition: inline


From: Peter Huewe <peterhuewe@gmx.de>

Trivial patch which adds the __init/__exit macros to the module_init/
module_exit functions of drivers/staging/p9auth/p9auth.c


Signed-off-by: Peter Huewe <peterhuewe@gmx.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
---
 drivers/staging/p9auth/p9auth.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/staging/p9auth/p9auth.c
+++ b/drivers/staging/p9auth/p9auth.c
@@ -334,6 +334,7 @@ static const struct file_operations cap_
 	.release = cap_release,
 };
 
+/* no __exit here because it can be called by the init function */
 static void cap_cleanup_module(void)
 {
 	int i;
@@ -360,7 +361,7 @@ static void cap_setup_cdev(struct cap_de
 		printk(KERN_NOTICE "Error %d adding cap%d", err, index);
 }
 
-static int cap_init_module(void)
+static int __init cap_init_module(void)
 {
 	int result, i;
 	dev_t dev = 0;
